### PR TITLE
Созонов Илья. Вариант 26. Технология MPI+OMP. Линейная фильтрация изображений (блочное разбиение). Ядро Гаусса 3x3.

### PR DESCRIPTION
**Pull request by sozozzya**

### CODE REVIEW

File: tasks/all/sozonov_i_image_filtering_block_partitioning/func_tests/func_all.cpp [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+    img[i * wdth] = 0.75;
+    img[(i * wdth) + wdth - 1] = 0.75;
+  }
+
+  return img;
+}
+
+}  // namespace sozonov_i_image_filtering_block_partitioning_all
+
+TEST(sozonov_i_image_filtering_block_partitioning_all, test_image_less_than_3x3) {
```
Comment (Ivan-Lysov) [inline tasks/all/sozonov_i_image_filtering_block_partitioning/func_tests/func_all.cpp:29]: add a test for an empty image

**Review by Ivan-Lysov — COMMENTED** [review]
add a test for an empty image

**Review by Ivan-Lysov — APPROVED** [review]
